Configure

```diff
@@ -132,15 +133,19 @@ my $gcc_devteam_warn = "-DDEBUG_UNUSED"
         . " -Werror"
         ;
 
-# These are used in addition to $gcc_devteam_warn when the compiler is clang.
+# In addition to the warning options above, select compilers get their own
+# set of additional warning options.  Use a compiler specific predefined
+# macro as index.
+our %devteam_additional_warn = ();
```

Comment:
... and to align it with previous comment, why not $additional_strict_warnings?

Reply:
Sure